### ISSUE TITLE
Feature: detect invalid use of working-directory option with action

### ISSUE THREAD

**Issue by chrisRedwine** [rhysd/actionlint#207]
The `working-directory` option can only be used in run steps, but not in actions (i.e., not with `uses`).

For example, the following workflow should fail analysis:

```yaml
on:
  workflow_dispatch:
jobs:
  foo:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v3
        working-directory: foo/
```

But it currently [succeeds](https://rhysd.github.io/actionlint/#eJw9jkEKwzAMBO95xX7A5NCbToU+pDiO07hJpWBJDfl9cRt6WtgZ2BWmDtilLtMq+30sukVLM3VPGbShSaQFUJ01CBN8cDYPa7Ss9kVqedOfBQRwfGXCbc5pEbezBlyzEmKyIqx9OvH1ffkb7UbhRxhLzcmkHtTm+w+EDDMr).

I think it'd be useful to add this check. Thanks for the great tool, btw!

**Comment by rhysd** [comment]
Thanks. It was bug of actionlint.

**Comment by rhysd** [comment]
This was fixed in [v1.6.17](https://github.com/rhysd/actionlint/releases/tag/v1.6.17).
